server/user.go

```diff
@@ -1157,36 +1156,25 @@ func garbageCollectUsers(period time.Duration, blockSize, minAccountAgeHours int
 	go func() {
 		// Add some randomness to the tick period to desynchronize runs on cluster nodes:
 		// 0.75 * period + rand(0, 0.5) * period.
-		period = (period >> 1) + (period >> 2) + time.Duration(rand.Intn(int(period>>1)))
+		period = period - (period >> 2) + time.Duration(rand.Intn(int(period>>1)))
```

Comment:
This is `0.5 * period + rand(0, 0.5) * period`. It's guaranteed to be less than `period`.
Mean = 0.75 * period, stddev = 0.14 * period.
Ideally we should make mean = period which the previous distribution did.

Reply:
The result is a uniformly distributed value [0.75 * period, 1.25 * period]:
```
period = period - (period >> 2) + time.Duration(rand.Intn(int(period>>1)))
```
==
```
1 - 0.25 + rand(0, 0.5)
```

The `rand.Intn(int(period>>1))` is _uniformly_ not _normally_ distributed. It has a math expectation of 0.25 * period.

Reply:
`(period >> 1) + (period >> 2) === period - (period >> 2)`
`0.5 + 0.25 === 1 - 0.25`

Reply:
My bad. You are right. The shift confused me.